unity/Holo/Assets/Scripts/ModelWithPlate.cs

```diff
@@ -9,12 +9,15 @@ public class ModelWithPlate : MonoBehaviour, IClickHandler
     public GameObject ButtonsModel;
     public GameObject ButtonsModelPreview;
     public GameObject PlateAnimated;
+    public GameObject modelClipPlane;
 

```

Comment:
We name public stuff, even fields, following `PascalCase`. So I would advice

```
public GameObject ModelClipPlane;
public GameObject CollectionButtons; // or ButtonsAdd
```

Why?
- I did not find C# coding standard that clarifies case of fields. The C# coding standards are found talk about public methods and properties, they should be `PascalCase`.
- So it seems that public fields make sense as `PascalCase` too. Then implementation of a class can switch between `FooBar` being a field to `FooBar` being a property, without breaking compatibility, since `FooBar` remains PascalCase in both situations.
- HoloToolkit also follows `PascalCase` for fields. See e.g. unity/Holo/Assets/HoloToolkit/Common/Scripts/Interpolator.cs .
- Note that Unity API uses lowercase `camelCase` for properties. This is contrary to C# standards I found and to HoloToolkit.

I do not really have a strong opinion about this, I'm fine with any coding standard, but we should follow it consistently in our code:)

Reply:
👍